examples/media/index.html

```diff
@@ -44,7 +45,44 @@
       <div class="header-content">
         <div class="searchbox-container">
           <label for="search-input">
-            <svg class="magnifier-icon" viewBox="0 0 17 17">
+            <svg
+              data-layout="mobile"
+              class="magnifier-icon"
+              xmlns="http://www.w3.org/2000/svg"
```

Comment:
`xmlns` (and `xmlns:xlink`) attributes are not required in HTML I think (I learned that recently).

Reply:
oh right, https://stackoverflow.com/a/18468348/11593190
good to know!

Reply:
Parcel failed to build it without `xmlns` :(